### ISSUE TITLE
Methods with unspecified or undefined record types

### ISSUE THREAD

**Issue by roger-mahler** [humlab-sead/sead_change_control#72]
Record type for method IRMS is not specified i.e. set to a NULL value in `tbl_methods`. The facet only uses inner joins on all routes in query which filters out these records.

```sql
SELECT tbl_methods.*
FROM tbl_dataset_masters
JOIN tbl_datasets USING (master_set_id)
JOIN tbl_methods USING (method_id)
LEFT JOIN tbl_record_types USING (record_type_id)
WHERE 1 = 1
  AND master_set_id = 11;
```
Two methods have a non-specified record type:

```sql
SELECT tbl_methods.*
FROM tbl_methods
WHERE record_type_id IS NULL;

175 "IRMS", "Isotope-ratio mass spectrometry"	
174 "AE", "Acid Extraction"
```
Closing this since it is a data error.

_Originally posted by @roger-mahler in https://github.com/humlab-sead/sead_query_api/issues/62#issuecomment-618252677_

**Comment by roger-mahler** [comment]
I guess "IRMS" should have record type "Isotopes"?

**Comment by visead** [comment]
We need Mats and or Chelsea to look at this.

**Comment by MGE4** [comment]
Yes, IRMS is a isotope analysis method, and should be under the umbrella term "Isotopes" in terms of record type.

**Comment by MGE4** [comment]
Im not sure if the "AE" method is used for Isotopes, Lipids or both... Chelsea might know this?
